invokeai/frontend/web/src/features/system/components/SettingsModal/SettingsUpsellMenuItem.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Flex, Icon, MenuItem, Text, Tooltip } from '@invoke-ai/ui-library';
+import { useTranslation } from 'react-i18next';
+import type { IconType } from 'react-icons';
+import { PiArrowUpBold } from 'react-icons/pi';
+
+export const SettingsUpsellMenuItem = ({ menuText, menuIcon }: { menuText: string; menuIcon: IconType }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Tooltip label={t('upsell.professionalUpsell')} placement="right" zIndex={2}>
+      <MenuItem as="a" href="http://invoke.com/pricing" target="_blank" icon={menuIcon({})}>
+        <Flex gap="1" alignItems="center">
+          <Text pb="2px">{menuText}</Text>
+          <Box>
+            <Icon
+              as={PiArrowUpBold}
+              sx={{
+                mt: '2px',
+                fill: 'invokeYellow.500',
+              }}
+            />
```

Comment:
Please use style props instead of `sx`. Instead of using margin, we can use a Flex with alignItems/justifyContent center.